exercises/00_final/precipitation_climatology.py

```diff
@@ -53,29 +58,30 @@ def plot_zonal(data):
     None
 
     """
-    zonal_pr = data["pr"].mean("lon", keep_attrs=True)
+    zonal_precipitation = data["precipitation"].mean("longitude", keep_attrs=True)
 
-    fig, ax = plt.subplots(nrows=4, ncols=1, figsize=(12, 8))
+    figure, axes = plot.subplots(nrows=4, ncols=1, figsize=(12, 8))
 
-    zonal_pr.sel(lat=[0]).plot.line(ax=ax[0], hue="lat")
-    zonal_pr.sel(lat=[-20, 20]).plot.line(ax=ax[1], hue="lat")
-    zonal_pr.sel(lat=[-45, 45]).plot.line(ax=ax[2], hue="lat")
-    zonal_pr.sel(lat=[-70, 70]).plot.line(ax=ax[3], hue="lat")
+    zonal_precipitation.sel(lat=[0]).plot.line(ax=axes[0], hue="latitude")
+    zonal_precipitation.sel(lat=[-20, 20]).plot.line(ax=axes[1], hue="latitude")
+    zonal_precipitation.sel(lat=[-45, 45]).plot.line(ax=axes[2], hue="latitude")
+    zonal_precipitation.sel(lat=[-70, 70]).plot.line(ax=axes[3], hue="latitude")
 
-    plt.tight_layout()
-    for axis in ax:
+    plot.tight_layout()
+    for axis in axes:
```

Comment:
Better. Why don't people do this as standard 😅

To quote @christopheredsall: "Do they charge you by the letter?"

Reply:
I wasn't aware of `plt` and `ax` being standard, and I feel a bit conflicted in that case. My normal policy is to go for the standard way, if it were something that wasn't as established as matplotlib then I'd argue for using nicer names, but I think if it's not what people are expecting when they use the library then it probably causes more confusion not to go with the convention.  As you say, might make an interesting discussion point. :)

Reply:
The `ax` I am less set on - I personally think `axis in axes` is a better way to write.
I'll let you decide what to go for.

Reply:
Great, will stick with that :)